### ISSUE TITLE
I want to know the specific parameter settings for HDBSCAN clustering algorithm

### ISSUE THREAD

**Issue by jinx-USTC** [zhangxinyu-xyz/PAST-ReID#4]
As your paper demonstrated, HDBSCAN clustering method outperforms DBSCAN, but when I use the commonly-used DBSCAN (from sklearn.cluster import DBSCAN) for clustering, I found that DBSCAN performs better than HDBSCAN (pip install HDBSCAN). I think maybe the specific parameters setting for HDBSCAN is vital, so I want to know the details of parameters setting about HDBSCAN in your paper, eg, min_cluster_size=?, min_samples=? (your paper use 10), metric='precomputed'...

**Comment by zhangxinyu-xyz** [comment]
@jinx-USTC We just set the min_samples=10, metric='precomputed', and other parameters are the same as the default values.
